crates/sats/src/algebraic_type.rs

```diff
@@ -14,47 +14,15 @@ use enum_as_inner::EnumAsInner;
 ///
 /// The type system unifies the concepts sum types, product types, and built-in
 /// primitive types into a single type system.
-///
-/// Below are some common types you might implement in this type system.
-///
-/// ```ignore
-/// type Unit = () // or (,) or , Product with zero elements
-/// type Never = (|) // or | Sum with zero elements
-/// type U8 = U8 // Builtin
-/// type Foo = (foo: I8) != I8
-/// type Bar = (bar: I8)
-/// type Color = (a: I8 | b: I8) // Sum with one element
-/// type Age = (age: U8) // Product with one element
-/// type Option<T> = (some: T | none: ())
```

Comment:
Why remove this comment? Does it have anything to do with SATS not having parametric types?